examples/run.py

```diff
@@ -61,4 +118,4 @@
     imagelab.list_possible_issue_types()
     issue_types = {issue_name: {}}
     imagelab.find_issues(issue_types)
-    imagelab.report([issue_name])
+    imagelab.report(issue_types=[issue_name])
```

Comment:
```suggestion
    imagelab.report(issue_types=[issue_name]+imagelab.list_possible_issue_types())
```

Reply:
or whatever is needed to simultaneously check for the custom new issue type along with the built-in default types